README.md

```diff
@@ -1 +1,44 @@
-# Mosha-Travel
\ No newline at end of file
+# KomiKult
+A comic characters list app that displays Marvel characters. It also allows you to Add your commit and like to a character. All data is preserved thanks to the external Marvel API.
+## Built With
+- HTML
+- CSS
+- JavaScript
+- Marvel Api, involvement Microverse Api
+- Webpack 
+## Live Demo
```

Comment:
- [ ] Please provide a visible link to the recorded video presenting the features of the project. Make sure the video contains a highlight about a piece of the code you find interesting or is proud of making. All members should appear in the video.

Reply:
Done!